Esri/arcgis-maps-sdk-swift-samples
# [New] Add KML layer

## Description

This PR implements `Add KML layer` in `Layers` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-AddKMLLayer/Shared/Samples/Add%20KML%20layer)

## Linked Issue(s)

- `swift/issues/6176`

## How To Test

- Use the picker to switch between sources to see the different KML layers displayed on the map.

## Screenshots

![add-kml-layer](https://github.com/user-attachments/assets/7c50fd6e-2954-4023-9075-ce8656f20d91)
